pkg/testutil/framework.go

```diff
@@ -307,6 +315,182 @@ func (f *Framework) SetupServiceCluster(ctx context.Context, cl *RecordingClient
 	return serviceCluster
 }
 
+func (f *Framework) KubeCarrierOperatorCheck(
+	ctx context.Context,
+	t *testing.T,
+	managementClient *RecordingClient,
+	managementScheme *runtime.Scheme,
+) {
+	namePrefix := constants.KubeCarrierDefaultName + "-operator"
+	namespaceName := constants.KubeCarrierDefaultNamespace
```

Comment:
@nmiculinic also, putting check functions in the framework can help we use the constants from the internal/constants packages, otherwise, we need to hardcode these constants in the e2e tests, having defaulting constants in 2 places is not really nice

Reply:
hmm....ok; that's a valid reason to stay withing `testutil`, but not withing framework. Having it a free function without framework seems a nicer solution long term.